src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java

```diff
@@ -148,6 +149,37 @@ public TestLinkAPI(URL url, String devKey) throws TestLinkAPIException {
 
         this.miscService.checkDevKey(devKey);
     }
+    
+    // add connection with a proxy

```

Comment:
Ah, just had time to see the error in Travis. @DanProgs could you add a Javadoc here, and check the trailing spaces here:

```
[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[152] (regexp) RegexpSingleline: Line has trailing spaces.

[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[154,5] (javadoc) JavadocMethod: Missing a Javadoc comment.

[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[159] (regexp) RegexpSingleline: Line has trailing spaces.
```